aspnetcore/fundamentals/minimal-apis/includes/parameter-binding8.md

```diff
@@ -110,13 +110,13 @@ To make `pageNumber` optional, define the type as optional or provide a default
 
 The preceding nullable and default value applies to all sources:
 
-[!code-csharp[](~/fundamentals/minimal-apis/7.0-samples/WebMinAPIs/Program.cs?name=snippet_op3)]
+:::code language="csharp" source="~/fundamentals/minimal-apis/7.0-samples/WebMinAPIs/Program.cs" id="snippet_op3" highlight="4":::
 
 The preceding code calls the method with a null product if no request body is sent.
 
 **NOTE**: If invalid data is provided and the parameter is nullable, the route handler is ***not*** run.
 
-[!code-csharp[](~/fundamentals/minimal-apis/7.0-samples/WebMinAPIs/Program.cs?name=snippet_op4)]
+:::code language="csharp" source="~/fundamentals/minimal-apis/7.0-samples/WebMinAPIs/Program.cs" id="snippet_op4" highlight="4":::
```

Comment:
Adde highlight for nullable type:

![image](https://github.com/dotnet/AspNetCore.Docs/assets/28659384/3d69b1a6-a9c2-4eca-aba9-fe8f2db66ba6)